script.js

```diff
@@ -1,4 +1,235 @@
+  // draw one new card for player and computer each
+  playerHand.push(shuffledDeck.pop())
+  computerHand.push(shuffledDeck.pop())
+  // generate another card for each player
+  playerHand.push(shuffledDeck.pop())
+  computerHand.push(shuffledDeck.pop())
+  if (playerHand.length == 2){ 
+    currentGameMode == GAME_MODE_HIT_OR_STAND
+    console.log(`game mode: ${currentGameMode}`)
+    console.log (`player hand ${printCards(playerHand)}, com hand: ${printCards(computerHand)}`)
+    myOutputValue = `${genericOutput}<br>
+    Please type 'hit' or 'stand' to decide if you wish to draw cards or end your turn.`
+    }
+//=======================Determine if player hits or stands=====================
+if (input.toLowerCase().includes('hit')){
+  currentGameMode = GAME_MODE_HIT_OR_STAND
+  currentPlayer = PLAYER
+  console.log(`input: ${input}, game mode: ${currentGameMode}`)
+  // draw one card for player and computer each
+  playerHand = dealCardToHand(deck)
+  // add new card to player's hand array
+  playerHand.push(deck.pop())
+  if (cardTotalSum > maxCardLimit){
+  gameOver = true;
+  myOutputValue = `${genericOutput}<br>Player went bust! Hit refresh to start again.`
+  }
+  else if (cardTotalSum < maxCardLimit)
+  myOutputValue = `${genericOutput}<br>Please type 'hit' or 'stand' to decide if you wish to draw cards or end your turn.`
+  }
+if (input.toLowerCase().includes('stand')){
+  currentGameMode = GAME_MODE_HIT_OR_STAND
+  console.log(`input: ${input}, game mode: ${currentGameMode}`)
+  currentPlayer = COMPUTER
+  myOutputValue = `${genericOutput}<br>Player chose to stand. Press submit for Computer's turn.`
+}
+return myOutputValue;
+}
+
+//=======================Determine winner=====================
+var winner = function (){
+  if (playerCard.rank > computerCard.rank
+    && input.toLowerCase().includes('stand')){
+    myOutputValue = `You win! Hit refresh to play again.`
+  } 
+  if (computerCard.rank > playerCard.rank
+    && input.toLowerCase().includes('stand')){
+    myOutputValue = `You lose! Hit refresh to play again.`
+  } 
+
+}
+
+//=======================Calculate sum total of cards=====================
+var calcHandTotal = function (hand){
+  var cardTotalSum = 0;
+  for (var i = 0; i < hand.length; i+=1){
+    var cardTotalSum = cardTotalSum + hand[i].value;
+  }
+  return cardTotalSum;
+}
+
+var playerHandTotal = 0;
+var comHandTotal = 0;
+console.log (`player total: ${playerHandTotal}, com total: ${comHandTotal}`)
+// generate total value of hands
+playerHandTotal = calcHandTotal(playerHand)
+computerHandTotal = calcHandTotal(computerHand)
+//=======================Blackjack logic=====================
+var blackJack = function (){
+  var myOutputValue = ''
+  //player has blackjack
+  if (playerHand.length == 2 
+    && cardTotalSum == maxCardLimit){
+      console.log('blackjack')
+    if (blackJack(playerHand)){
```

Comment:
There is no need for the if statement here since you have already determined that the player has a blackjack. Same for line 196.